tests/models/test_horovod.py

```diff
@@ -93,7 +95,7 @@ def test_horovod_cpu(tmpdir):
 @RunIf(skip_windows=True, horovod=True, skip_49370=True)
 def test_horovod_cpu_accumulate_grad_batches(tmpdir):
     trainer_options = dict(
-        default_root_dir=tmpdir,
+        default_root_dir=str(tmpdir),
```

Comment:
interesting. was this needed because of the json dump on the other side?

Reply:
Indeed, all other tests actually have this as well.